lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -615,11 +615,13 @@ public static function get_merged_data( $origin = 'custom' ) {
 		$result->merge( static::get_theme_data() );
 		if ( 'theme' === $origin ) {
 			$result->set_spacing_sizes();
+			$result->resolve_theme_file_uris();
```

Comment:
I've added these here for now because it ensures relative paths are resolved everywhere where merged data , e.g., [wp_get_global_styles](https://github.com/WordPress/wordpress-develop/blob/6974b994de5e9c707607dcb4233bd785865b4c6a/src/wp-includes/global-styles-and-settings.php#L128), [wp_get_global_stylesheet](https://github.com/WordPress/wordpress-develop/blob/6974b994de5e9c707607dcb4233bd785865b4c6a/src/wp-includes/global-styles-and-settings.php#L182), and by extension [get_block_editor_settings](https://github.com/WordPress/wordpress-develop/blob/6974b994de5e9c707607dcb4233bd785865b4c6a/src/wp-includes/block-editor.php#L511) or the [global styles controller REST API endpoint](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/rest-api/endpoints/class-wp-rest-global-styles-controller.php#L605).

If we wanted to make this transform more imperative, then it could be a separate static method in this class, or we add an `$options` param to `get_merged_data` or....